core/server/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -610,8 +609,12 @@ private void processWorkerRemovedBlocks(MasterWorkerInfo workerInfo,
     for (long removedBlockId : removedBlockIds) {
       MasterBlockInfo masterBlockInfo = mBlocks.get(removedBlockId);
       if (masterBlockInfo == null) {
-        LOG.warn("Worker {} removed block {} but block does not exist.", workerInfo.getId(),
-            removedBlockId);
+        LOG.warn("Worker {} informs the removed block {}, but block metadata does not exist"
```

Comment:
I'm not sure if this error message is completely correct. A worker can remove a block even if the master did not command it to (like eviction). Therefore, maybe the original message is more accurate.